models/quality_measures/intermediate/nqf0059_diabetes_hemoglobin_a1c/quality_measures__int_nqf0059_denominator.sql

```diff
@@ -0,0 +1,175 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
+   )
+}}
+
+with  visit_codes as (
+
+    select
+          code
+        , code_system
+    from {{ ref('quality_measures__value_sets') }}
+    where lower(concept_name) in (
+          'office visit'
+        , 'home healthcare services'
+        , 'preventive care services established office visit, 18 and up'
+        , 'preventive care services initial office visit, 18 and up'
+        , 'annual wellness visit'
+        , 'telephone visits'
+    )
+
+), visits_encounters as (
+    select patient_id
+         , coalesce(encounter.encounter_start_date,encounter.encounter_end_date) as min_date
+         , coalesce(encounter.encounter_end_date,encounter.encounter_start_date) as max_date
+    from {{ref('quality_measures__stg_core__encounter')}} encounter
+    inner join {{ref('quality_measures__int_nqf0059__performance_period')}} as pp
+        on coalesce(encounter.encounter_end_date,encounter.encounter_start_date) >= pp.performance_period_begin
+        and  coalesce(encounter.encounter_start_date,encounter.encounter_end_date) <= pp.performance_period_end
+    where lower(encounter_type) in (
```

Comment:
Can you add 'telehealth' to this list? That's the equivalent of 'telephone visits' from the measure's value sets.

Reply:
`telehealth` has been added